_attributes/common-attributes.adoc

```diff
@@ -278,17 +278,28 @@ endif::[]
 :ocid: OCID
 :ocvs-first: Oracle(R) Cloud VMware Solution (OCVS)
 :ocvs: OCVS
-// Cluster Observability Operator
-:coo-first: Cluster Observability Operator (COO)
-:coo-full: Cluster Observability Operator
-:coo-short: COO
-//ODF
-:odf-first: Red Hat OpenShift Data Foundation (ODF)
-:odf-full: Red Hat OpenShift Data Foundation
-:odf-short: ODF
-:rh-dev-hub: Red Hat Developer Hub
-//IBU
-:lcao: Lifecycle Agent
+// Red Hat OpenStack Platform (RHOSP)/OpenStack
```

Comment:
This might need to be {rh-openstack} - not sure what the ruling is for acronyms. :) I'll check to make sure.

Reply:
This is just a comment in the attributes file, it will never render anywhere. Solely for writer reference as to what these are :)

Reply:
Ah ok! Ty for clarifying that :)